metadata.py

```diff
@@ -40,15 +40,27 @@ def appmetadata() -> AppMetadata:
         # analyzer_license="",  # short name for a software license
     )
     # and then add I/O specifications: an app must have at least one input and one output
-    metadata.add_input(AnnotationTypes.TimeFrame)
+    metadata.add_input(AnnotationTypes.TimeFrame)  # Output from SWT
+    metadata.add_input(DocumentTypes.TextDocument) # Output from OCR
     metadata.add_output(DocumentTypes.TextDocument)
+    metadata.add_output(AnnotationTypes.Alignment)
     
-    # (optional) and finally add runtime parameter specifications
-    metadata.add_parameter(name='a_param', description='example parameter description',
-                           type='boolean', default='false')
-    # metadta.add_parameter(more...)
     
-    # CHANGE this line and make sure return the compiled `metadata` instance
+    # (optional) and finally add runtime parameter specifications
+    metadata.add_parameter(name='contain_labels',
+                           description=
+                           '''
+                           A list of labels of TimeFrame to look for.
+                           Available options are:
+                           ['bars', 'tones', 'bars-and-tones','speech',
+                           'noise', 'music', 'slate', 'chyron',
+                           'lower-third', 'credits']
+                           User needs to provide a string of comma-separated labels, 
+                           e.g. 'bars, credits, chyron'.
+                           Default is all are selected.
```

Comment:
If you really want to give a predefine choices, you can use `choices=['x', 'y', 'z']` argument. But in this case, since we can come up with a different name of the frame label in the future, I don't think it's safe to limit the filter names here.